secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret
+type: Opaque
+data:
+  secret_key:  TVlfU0VDUkVUX0tFWQ== # is encoded version of MY_SECRET_KEY
+  client_id:  NDYxNjQwZmQxMDAwNjQyZDA0NjI=
+  client_secret:  YTFhMmU4ZjFkYzMxMDM1MWQxOGI0YTNhMzY0OTNjN2VkYzljMGUyMg==
+  mongo_client:  bW9uZ29kYjovL21vZHVsZTEwOnQwcXd4VTNWN01aMlk1NW1oYm5SQXRlZmlHcndKVFVIYVRPNk1oTzBLdjhLVFczUFg2R0t2bG9uM0xGMWg4dTI5QkNnUVJZRTQxd2dBQ0RiU1R6c2dnPT1AbW9kdWxlMTAubW9uZ28uY29zbW9zLmF6dXJlLmNvbToxMDI1NS8/c3NsPXRydWUmcmVwbGljYVNldD1nbG9iYWxkYiZyZXRyeXdyaXRlcz1mYWxzZSZtYXhJZGxlVGltZU1TPTEyMDAwMCZhcHBOYW1lPUBtb2R1bGUxMEA=
+  LOGGLY_TOKEN:  NTI4NmEwNGYtMTZhYy00ZjA1LTgzN2YtNGY3MTJkOWM4MDk1
+  LOG_LEVEL:  REVCVUc=
```

Comment:
These are only encoded, not encrypted, so they're now all public knowledge!

We should remove this file (add it to .gitignore, and remove the existing memory with `git rm --cached secrets.yaml`), although it is nice to leave a template version that doesn't have the secrets, i.e. we could have a new file called `secrets.yaml.template` that just has:
```
apiVersion: v1
kind: Secret
metadata:
  name: secret
type: Opaque
data:
  secret_key:  
  client_id:  
  client_secret:  
  mongo_client: 
  LOGGLY_TOKEN:  
  LOG_LEVEL:  
```

Reply:
done :)